tests/test_memory_core/build_tb.py

```diff
@@ -2140,6 +2140,33 @@ def software_gold(app_name, matrix_tmp_dir, give_tensor=False, print_inputs=None
         output_matrix = numpy.maximum(0, (numpy.matmul(b_mat, c_mat_trans, dtype=numpy.int16, casting='unsafe')))
         output_format = "CSF"
         output_name = "X"
+    
+    elif "concat_matmul.gv" in app_name:
+        # matrix b is completely dense
+        b_mat = get_tensor(input_name='B', shapes=[shapes_[0], shapes_[1]], give_tensor=give_tensor, tmp_dir=matrix_tmp_dir,
```

Comment:
For the input matrix shapes, we are not using randomized shapes anymore, refer to other test cases and set the shapes accordingly

Reply:
Also sparsity is no longer randomized, refer to other test cases and fix it

Reply:
done